### DISCUSSION TITLE
Issue in installing latest version

### DISCUSSION THREAD

**Discussion by Bhumika1997** [chaos-mesh/chaos-mesh#2415] [General]
When we are trying to install latest version of Chaos Mesh it is giving us error that few things alreay exist but we have uninstalled previous version and everything for chaos mesh then again started installing latest version but it is still giving us this error
can somebody please help us on this?
![image](https://user-images.githubusercontent.com/33874082/138811123-523b14f7-bebb-4c3d-9313-4a46728ac92b.png)

**Comment by STRRL** [comment]
I think you are using `install.sh` to install the Chaos Mesh, and you do not uninstall it cleanly.

If you only remove the namespace for Chaos Mesh, there are some cluster-level resources about chaos-mesh:
- customresourcedefinitions with group `chaos-mesh.org`, the names might be `<something>Chaos`, `Schedule`, `Workflow` and `WorkflowNode`
- validatingwebhookconfigurations `chaos-mesh-validation` and `validate-auth`
- mutatingwebhookconfigurations `chaos-mesh-mutation`
- clusterrole `chaos-mesh-chaos-controller-manager-cluster-level` and `chaos-mesh-debug-chaos-controller-manager-target-namespace`
- clusterrolebinding `chaos-mesh-chaos-controller-manager-cluster-level` and `chaos-mesh-debug-chaos-controller-manager-target-namespace`

You could remove these resources manually for purging Chaos Mesh installation.

At last, we suggest using helm to install and uninstall Chaos Mesh. `install.sh` could play well with `kind` and `minikube` clusters because you could easily purge and rebuild the whole cluster as you want. :)
